.changeset/large-showers-leave.md

```diff
@@ -0,0 +1,5 @@
+---
+"@biomejs/biome": patch
+---
+
+Fixed [4677](https://github.com/biomejs/biome/issues/4677). Now the rule `noUnusedImports` rule won't produce diagnostics for types used in comment of JS_EXPORT
```

Comment:
To describe changes to end users of Biome, let's avoid using internal names such as JS_EXPORT.

```suggestion
Fixed [#4677](https://github.com/biomejs/biome/issues/4677): Now the `noUnusedImports` rule won't produce diagnostics for types used in JSDoc comment of exports.
```